lib/domain/usecase/room/chat_get_pinned_events_interactor.dart

```diff
@@ -9,12 +9,17 @@ import 'package:matrix/matrix.dart';
 class ChatGetPinnedEventsInteractor {
   Stream<Either<Failure, Success>> execute({
     required Room room,
+    List<Event?>? eventsInMem,
```

Comment:
`eventsInMem` is events in memory right?

Reply:
Why do you need to include it?

Reply:
the previous idea is update room without needing to call `getPinnedEvents`, but because of room get updated through sync very quick, so this can be removed